rfcs/0136-stabilize-incrementally.md

```diff
@@ -0,0 +1,386 @@
+---
+feature: stabilize-incrementally
+start-date: 2022-09-15
+author: John Ericson
+co-authors: (find a buddy later to help out with the RFC)
```

Comment:
Is this still supposed to say “(find a buddy later to help out with the RFC)” even though it’s been merged?